samples/azure-quantum/check-ghz/CheckGHZ.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Integrated Hybrid: Blending classical and quantum computing\n",
+        "In this sample, you will discover how to blend classical and quantum instructions in the same program, all fully processed by the quantum computing back-end.\n",
+        "You can learn more how to program with Integrated hybrid at https://aka.ms/AQ/Hybrid/Docs\n",
```

Comment:
This link gives me a 404, which might be fine if we are going to make the link available before this goes live.